include/AudioEngine.h

```diff
@@ -26,6 +26,11 @@
 #define AUDIO_ENGINE_H
 
 #include <QMutex>
+
+#if (QT_VERSION >= QT_VERSION_CHECK(5,15,0))
```

Comment:
Yeah, the docs do say 5.14. I think it was because they did not say anything about using ``QRecursiveMutex`` and what version it was introduced when I made this PR. At the time, I went into the source of Qt and found this:

```
#if QT_DEPRECATED_SINCE(5,15)
    enum RecursionMode { NonRecursive, Recursive };
    QT_DEPRECATED_VERSION_X(5, 15, "Use QRecursiveMutex instead of a recursive QMutex")
    explicit QMutex(RecursionMode mode);

    QT_DEPRECATED_VERSION_X(5, 15, "Use QRecursiveMutex instead of a recursive QMutex")
    bool isRecursive() const noexcept
    { return QBasicMutex::isRecursive(); }
#endif
```

So I ended up using 5.15. I'll bump it down to 5.14 to align with what the docs say.